game/outfits.rpy

```diff
@@ -1286,14 +1274,26 @@ init -1 python in jn_outfits:
         is_jn_wearable=True
     ))
     __register_wearable(JNClothes(
-        reference_name="jn_clothes_square_bra",
-        display_name="Square bra",
+        reference_name="jn_clothes_striped_off_shoulder_sweater_bee",
+        display_name="Off-shoulder sweater bee",
+        unlocked=True,
+        is_jn_wearable=True
+    ))
+    __register_wearable(JNClothes(
+        reference_name="jn_clothes_striped_off_shoulder_sweater_autumn",
+        display_name="Off-shoulder sweater autumn",
+        unlocked=True,
+        is_jn_wearable=True
+    ))
+    __register_wearable(JNClothes(
+        reference_name="jn_clothes_striped_off_shoulder_sweater_creamsicle",
+        display_name="Off-shoulder sweater creamsicle",
         unlocked=False,
         is_jn_wearable=True
     ))
     __register_wearable(JNClothes(
-        reference_name="jn_clothes_striped_off_shoulder_sweater",
-        display_name="Off-shoulder sweater",
+        reference_name="jn_clothes_striped_off_shoulder_sweater_nightbloom",
+        display_name="Off-shoulder sweater nightbloom",
         unlocked=False,
         is_jn_wearable=True
     ))
```

Comment:
I think with these, it might make a little more sense if we have the descriptor (like "Nightbloom") ahead of the sweater part of the name! For example:

- Autumn off-shoulder sweater
- Nightbloom off-shoulder sweater
- Cream off-shoulder sweater

I think this way, it sounds a little more like how you'd see it displayed in a store, or if you were shopping online for it.

Also, I think if we have multiple sweaters that are striped, we probably don't need to identify that each time! We can just go by the main identifying part of each sweater.

So if we take the nightbloom sweater:

- For the reference name, I'd suggest "jn_clothes_nightbloom_off_shoulder_sweater"
- For the display name, I'd suggest "Nightbloom off-shoulder sweater"

Reply:
As before - just make sure any changes in code to reference names match the folders!